Task6/helloworld/tests/test_helloworld.py

```diff
@@ -0,0 +1,5 @@
+from helloworld import __version__
+
```

Comment:
We need a unit test of the app method. Hint: use TestClient https://fastapi.tiangolo.com/tutorial/testing/